webapp/ember-cli-build.js

```diff
@@ -21,7 +21,7 @@ module.exports = function(defaults) {
           'handlebars.js': null
       },
       'ember-cli-babel': {
-          includePolyfill: not_testing,
```

Comment:
@stefanpenner for example https://travis-ci.org/getslash/backslash/builds/232506426#L6052